src/game-engine/net/inbound-packets/item-on-object-packet.js

```diff
@@ -39,7 +40,22 @@ const itemOnObjectPacket = (player, packet) => {
         return;
     }
 
-    const objectConfig = filestore.configStore.objectStore.getObject(objectId);
+    let objectConfig = filestore.configStore.objectStore.getObject(objectId);
+    if (objectConfig.configChangeDest) {
```

Comment:
Since this is player related, we could move line 44 to 58 to the player class, like I did for NPCs:
https://github.com/runejs/server/pull/298/files#diff-506bad9e56d4825809f1403f592854ac37e24e2dedc00f7d0d6201f609c824aeR938